src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -55,11 +60,36 @@ const AttributesAccordion = (props: Props) => {
     props.setAttributesAccordionExpandedPanel(newExpandedPanels[0]);
   };
 
+  const onReset = () => {
+    props.resetSelectedAttributes();
+    customDownloadStorageService.saveSelectedAttributes({});
+  };
+
+  const buildSection = (
+    section: string,
+    hideTitles?: boolean,
+    hideUnchecked?: boolean
+  ) => {
+    return (
+      <AttributesAccordionSection
+        section={section}
+        hideTitles={hideTitles}
+        hideUnchecked={hideUnchecked}
+      />
+    );
+  };
   return (
     <div className={styles.wrapper}>
       <div className={styles.dataSelectorHint}>
         Select the information you would like to download - these attributes
         will be displayed as columns in a table
+        <span className={styles.resetIcon} onClick={onReset}>
+          <ImageButton
+            buttonStatus={ImageButtonStatus.ACTIVE}
+            description={'Reset attributes'}
+            image={ResetIcon}
+          />
+        </span>
       </div>
       <Accordion
         preExpanded={Array(1).fill(props.expandedPanel)}
```

Comment:
`Array(1).fill(props.expandedPanel)` -> `[props.expandedPanel]` ?

Reply:
No idea what I was trying to do here 😄  I've updated it.